kevinzs/day02/day02.py

```diff
@@ -0,0 +1,20 @@
+def solver(input):
+    res = 0
+    for row in input:
+        min_value = min(int(x) for x in row)
+        max_value = max(int(x) for x in row)
+        res += max_value - min_value
+    return res
+
+def solver2(input):
+    res = 0
+    for row in input:
+        for x in range(len(row)):
+            for y in range(x+1, len(row)):
+                if int(row[x]) % int(row[y]) == 0:
+                    res += int(row[x]) / int(row[y])
+                    continue
+                elif int(row[y]) % int(row[x]) == 0:
+                    res += int(row[y]) / int(row[x])
+                    continue
```

Comment:
This is a really nice solution as you don't make more iterations than needed.

Seeing this code now makes me realize thay I may do lots of calculations that could be avoided 😆 Anyway, I do prefer a more declarative solution than working with indexes and breaking loops.

Really good job 💯